integration-cli/docker_cli_exec_test.go

```diff
@@ -67,7 +68,7 @@ func (s *DockerSuite) TestExecInteractive(c *check.C) {
 
 func (s *DockerSuite) TestExecAfterContainerRestart(c *check.C) {
 	testRequires(c, DaemonIsLinux)
-	out, _ := runSleepingContainer(c, "-d")
+	out, _ := runSleepingContainer(c)
```

Comment:
The `"-d"` here won't change anything but is not strictly required (automatically added by `runSleepingContainer`).